lib/pdc/resource/associations/builder.rb

```diff
@@ -0,0 +1,44 @@
+require 'active_support/dependencies'
```

Comment:
Isn't this addition the same as the one in #47 ? Yet I can't see them being re-used. Any duplication going on or?

Reply:
@romanofski Yes. That is duplication. @xiangge and I are all base on the master code and implement has_many and belongs_to respectively. We will rebase it after one patch is merged.

Reply:
Ah I see. Fair enough.